js_modules/dagster-ui/packages/ui-core/src/app/ContentRoot.tsx

```diff
@@ -8,6 +8,7 @@ const WorkspaceRoot = lazy(() => import('../workspace/WorkspaceRoot'));
 const OverviewRoot = lazy(() => import('../overview/OverviewRoot'));
 const FallthroughRoot = lazy(() => import('./FallthroughRoot'));
 const AssetsCatalogRoot = lazy(() => import('../assets/AssetsCatalogRoot'));
+const AssetsOverview = lazy(() => import('../assets/AssetsOverview'));
```

Comment:
Not used? Does this need a route added to the `Switch`?

Reply:
Ah good catch, this will only be displayed on cloud so this line isn't needed